src/Categories/Functor/CartesianClosed.agda

```diff
@@ -0,0 +1,37 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Functor.CartesianClosed where
+
+open import Level
+
+open import Categories.Category.CartesianClosed.Bundle using (CartesianClosedCategory)
+open import Categories.Category.CartesianClosed
+open import Categories.Functor using (Functor; _∘F_)
+open import Categories.Functor.Cartesian
+
+import Categories.Morphism as M
+
+private
+  variable
+    o ℓ e o′ ℓ′ e′ : Level
+
+record IsCartesianClosedF (C : CartesianClosedCategory o ℓ e) (D : CartesianClosedCategory o′ ℓ′ e′)
+  (F : Functor (CartesianClosedCategory.U C) (CartesianClosedCategory.U D)) : Set (o ⊔ ℓ ⊔ e ⊔ o′ ⊔ ℓ′ ⊔ e′) where
+  private
+    module C = CartesianClosedCategory C
```

Comment:
Because of how Agda works, this causes a huge amount of bloat. Best to use `using` here too, so that the resulting module have a small interface.